src/openrct2/core/Path.hpp

```diff
@@ -16,9 +16,9 @@
 namespace Path
 {
     utf8* Append(utf8* buffer, size_t bufferSize, const utf8* src);
-    std::string Combine(const std::string& a, const std::string& b);
+    std::string Combine(const std::string_view& a, const std::string_view& b);
```

Comment:
It shouldn't be necessary to do `const std::string_view&` IIRC.

```suggestion
    std::string Combine(std::string_view a, std::string_view b);
```